tensorboardX/record_writer.py

```diff
@@ -66,7 +66,7 @@ def __del__(self):
         self.close()
 
     def bucket_and_path(self):
-        bp = self.path.split("/")
+        bp = self.path.split("/")[2:]
```

Comment:
Thank you for this fix! You're right - I missed this. Do you mind being more safe on this and change to:

```
path = self.path
if path.startswith("s3://"):
        path = path[len("s3://") :]
bp = path.split("/")
...
```

Reply:
good call, will do!